src/content/ja/2022/css.md

```diff
@@ -0,0 +1,1347 @@
+
+{{ figure_markup(
+    image="rules-per-page.png",
+    caption="ページごとのスタイルルールの総数の分布。",
+    description="ページあたりのスタイルルールの数の10、25、50、75、90パータイルを示す棒グラフ。モバイルページとデスクトップページは非常に似ている傾向があります。モバイルの値は、1ページあたり52、224、613、1,197、2,023のルールです。",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSPMUe75uC8laUvzfECAPpT9fPoTdZA6FYDULorsUzKVmLmagphzh1UoXRcmyd6a3gxqr6dxIhmJyv5/pubchart?oid=2137701589&format=interactive",
+    sheets_gid="1977925185",
+    sql_file="rules_per_stylesheet.sql"
+  )
+}}
+
+ページ内のスタイルルールの数を見ると、すべてのパーセンタイルで増加しており、低いパーセンタイルほどモバイル向け、高いパーセンタイルほどデスクトップ向けのルールが多くなっています。この増加幅は相当なものです。50パーセンタイルではデスクトップのルールが130個、90パーセンタイルでは202個増えています。
+
+{{ figure_markup(
+    image="rules-per-stylesheet.png",
+    caption="スタイルシートごとのルール数の分布。",
+    description="スタイルシートごとのルール数の10、25、50、75、90パーセンタイルを示す棒グラフ。モバイルとデスクトップの値はほぼ同じです。モバイルページでは、スタイルシートあたりのルール数が0、4、31、110、285となっています。",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSPMUe75uC8laUvzfECAPpT9fPoTdZA6FYDULorsUzKVmLmagphzh1UoXRcmyd6a3gxqr6dxIhmJyv5/pubchart?oid=56198269&format=interactive",
+    sheets_gid="1977925185",
+    sql_file="rules_per_stylesheet.sql"
+  )
+}}
+
+読み込まれたスタイルシートの総数から、通常、人々はCSSを複数のスタイルシートに分割していることがわかります。50パーセンタイルでは、スタイルシートごとに31ルールが読み込まれ、90パーセンタイルでは、デスクトップで276ルール、モバイルで285ルールにまで増加します。
+
+## セレクターとカスケード
+
+2022年には、[`@layer`](https://developer.mozilla.org/ja/docs/Web/CSS/@layer)がすべてのエンジンに搭載され、カスケードに関しても変化がありました。この新しいアットルールは、セレクターをレイヤーにグループ化し、そのレイヤーの優先順位を管理できます。
+
+この新しいカスケード管理方法が広く使われるようになるにはまだ少し早いですが、セレクターの使い方がどのように進化してきたかを見てみましょう。
+
+### クラス名
+
+{{ figure_markup(
+    image="top-selector-classes.png",
+    caption="ページ数の割合でもっとも普及しているクラス名です。",
+    description="もとも多くのページで使用されているCSSクラス名を棒グラフで表示。モバイルとデスクトップは同じような結果になっています。モバイルでは、クラス名のトップは`active`で、47%のページで使用されています。次に`fa`が33%、`fa`をプレフィックスとする他のクラスが32%、`wp`をプレフィックスとするクラスが31%となっています。残りの上位10クラスは、降順に並べると次のようになります。`button`が27%で `adoption`、 `pull-right`、 `emoji`、 `disabled` が26%、最後に `pull-left` と `title` が25%となっています。",
+    chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSPMUe75uC8laUvzfECAPpT9fPoTdZA6FYDULorsUzKVmLmagphzh1UoXRcmyd6a3gxqr6dxIhmJyv5/pubchart?oid=1082092053&format=interactive",
+    height="691",
+    sheets_gid="1020483137",
+    sql_file="top_selector_classes_wp_fa_prefixes.sql"
+  )
+}}
+
+2020年、2021年と同様、ウェブ上でもっとも普及しているクラス名は`active`です。Font Awesomeのプレフィックスである`fa`、`fa-*`は依然として2位と3位を占めています。しかし、`wp-*`クラス名は順位を上げ、4位となりました。2021年には20％だったのが、現在では31％のページで表示されています。また、`has-large-font-size`といったクラス名も登場しており、これらは新しいWordPressブロックエディターで使用されています。
+
+トップ20から`clearfix`が消え、今では10%のページで見られるだけです。これは、フロートベースのレイアウトがウェブから消えつつあることを明確に示しています。
+
+{{ figure_markup(
+    image="top-selector-ids.png",
+    caption="ページ数の割合でもっとも普及しているID名です。",
+    description="もっとも多くのページで使用されているCSS IDを示す棒グラフ。モバイルとデスクトップの傾向は似ています。IDコンテンツは15%のページで使用され、次にフッターが12%、ヘッダー、`fb-root`、`fb_dialog_loader_close`、`fb_dialog_ipad_overlay`、`fb_dialog_loader_spinner`が10%、レスポンスとコメントが9%、そして最後にメインが8%のページで使用されていることがわかります。",
```

Comment:
Could you double check this one as think you've translated some of the code words.

Here's the English version:

```
    description="Bar chart showing the CSS IDs used on the most pages. Mobile and desktop trends are similar. The ID `content` is used on 15% of pages, followed by `footer` on 12%, `header`, `fb-root`, `fb_dialog_loader_close`, `fb_dialog_ipad_overlay`, and `fb_dialog_loader_spinner` all on 10%, then `respond` and `comments` on 9%, and finally `main` on 8% of pages.",
```

Reply:
So why does `footer` and `header` not show as code words in your translation?

Reply:
@tunetheweb  Thanks I translated